Environment_Engine/Compute/ExtrudeToVolume.cs

```diff
@@ -35,28 +35,22 @@
 using BH.oM.Reflection.Attributes;
 using System.ComponentModel;
 
+using BH.oM.Analytical.Elements;
+
 namespace BH.Engine.Environment
 {
     public static partial class Compute
     {
-        [Description("Takes an Architecture Room with a Floor Perimieter and creates a collection of Environment Panels which represent the closed volume of the room. The name of the room becomes the connected space for the panels")]
-        [Input("room", "An Architecture Room with a floor perimeter to extrude into a collection of panels")]
-        [Input("height", "The height of the room, as a double, to calculate the ceiling level of the room. This will be used as the Z value of the perimeter + the given height")]
-        [Output("panels", "A collection of Environment Panels which represent the closed volume of the room")]
-        public static List<Panel> ExtrudeToVolume(this Room room, double height)
-        {
-            Polyline floor = room.Perimeter.ICollapseToPolyline(BH.oM.Geometry.Tolerance.Angle);
-            return floor.ExtrudeToVolume(room.Name, height);
-        }
-
-        [Description("Takes an Environment Space with a Floor Perimieter and creates a collection of Environment Panels which represent the closed volume of the space. The name of the space becomes the connected space for the panels")]
-        [Input("space", "An Environment Space with a floor perimeter to extrude into a collection of panels")]
-        [Input("height", "The height of the room, as a double, to calculate the ceiling level of the space. This will be used as the Z value of the perimeter + the given height")]
-        [Output("panels", "A collection of Environment Panels which represent the closed volume of the space")]
-        public static List<Panel> ExtrudeToVolume(this Space space, double height)
+        [Description("Takes a region with a Floor Perimieter and creates a collection of Environment Panels which represent the closed volume of the region. The name of the region becomes the connected space for the panels")]
+        [Input("region", "A region with a floor perimeter to extrude into a collection of panels")]
+        [Input("height", "The height of the region, as a double, to calculate the ceiling level of the region. This will be used as the Z value of the perimeter + the given height")]
+        [Output("panels", "A collection of Environment Panels which represent the closed volume of the region")]
```

Comment:
```suggestion
        [Input("height", "The height of the region, as a double, to calculate the ceiling level of the region. This will be used as the Z value of the perimeter + the given height")]
        [Input("tolerance", "The degree of tolerance on the angle calculation for collapsing the regions perimeter to a polyline. Default is equal to BH.oM.Geometry.Tolerance.Angle.")]
        [Output("panels", "A collection of Environment Panels which represent the closed volume of the region")]
```